### ISSUE TITLE
Get this immediately on open of PowerShell.

### ISSUE THREAD

**Issue by dotConsultant** [PowerShell/PSReadLine#2992]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.19043
BufferWidth: 120
BufferHeight: 30

Last 0 Keys


```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```
```


### Screenshot

After opening PowerShell got this exception with prompt to report bug. This was updated recently according to Windows Store update history. Was working well earlier in week.

### Environment data

```console
Can't seem to run anything at the prompt. Won't let me.
```

### Steps to reproduce

Open PowerShell in Windows Terminal.

### Expected behavior

Ability to run commands.

### Actual behavior

Error. Can't seem to run anything in PowerShell.

**Comment by frandi** [comment]
I got this same error after updating to PowerShell 7.2.0, waiting for the fix.

**Comment by nickalbrecht** [comment]
I got the same error. Though my version of PSReadLine is `2.2.0-beta2`. And for some reason it says I'm on windows 10, but I'm actually on Windows 11.

### Environment
PSReadLine: 2.2.0-beta2
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.22000
BufferWidth: 120
BufferHeight: 30

Last 0 Keys
```

```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```

**Comment by skyhammer** [comment]
I initially ran into this problem too, but was able to resolve it by updating the module, as per the readme. The steps I used to update are as follows:
1. Open PowerShell ISE as administrator
2. Type:  `& 'C:\Program Files\PowerShell\7\pwsh.exe' -noprofile -command "Install-Module PSReadLine -Force -SkipPublisherCheck -AllowPrerelease"` - NOTE: You may need to change the PowerShell install path as mine is the 'default' location.

**Comment by qmmm61** [comment]
i got same error, my powershell installed by windows store, and i install module PSReadLine manually before , the version is `2.2.0-beta2`, i find the module in directory `C:\Users\username\Documents\PowerShell\Modules`, get module directory by `$env:PSModulePath`, i find powershell already have it ,the version is `2.1.0`, so i just remove the PSReadLine by me installed manual in directory. then, open powershell with no error. i install the PSReadLine manual again, the version is ` 2.2.0-beta4`, reopen powershell is still fine, i think the error in PSReadLine version `2.2.0-beta2`, upgrade or degrade the version can resolve it.

**Comment by daxian-dbw** [comment]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.

**Comment by dotConsultant** [comment]
> i got same error, my powershell installed by windows store, and i install module PSReadLine manually before , the version is `2.2.0-beta2`, i find the module in directory `C:\Users\username\Documents\PowerShell\Modules`, get module directory by `$env:PSModulePath`, i find powershell already have it ,the version is `2.1.0`, so i just remove the PSReadLine by me installed manual in directory. then, open powershell with no error. i install the PSReadLine manual again, the version is ` 2.2.0-beta4`, reopen powershell is still fine, i think the error in PSReadLine version `2.2.0-beta2`, upgrade or degrade the version can resolve it.

I followed your advice and all is good now. Will have to follow others advise and reinstall PSReadLine if I need it.
